dts/bindings/base/zephyr,memory-attr.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2023, Carlo Caione <ccaione@baylibre.com>
+# SPDX-License-Identifier: Apache-2.0
+
+properties:
+  zephyr,memory-attr:
+    type: string
+    enum:
+      - "RAM"
+      - "RAM_NOCACHE"
+      - "FLASH"
+      - "PPB"
+      - "IO"
+      - "EXTMEM"
+    description: |
+      Attribute for the memory region.
+
+  reg:
+    required: true
```

Comment:
shouldn't this binding include `base.yaml` instead?

Reply:
yes, this is still required though.